apps/mobile-wallet/src/features/addressesManagement/AddressDetailsModal.tsx

```diff
@@ -61,34 +59,32 @@ const AddressDetailsModal = withModal<AddressDetailsModalProps>(({ id, addressHa
     <BottomModal modalId={id} title={<AddressBadge addressHash={addressHash} fontSize={16} />}>
       <Content>
         <RoundedCard>
-          <AnimatedBackground shade={address?.settings.color} isAnimated />
+          <AddressAnimatedBackground addressHash={addressHash} />
```

Comment:
Now, only the animated bg color will rerender on settings changes, not the whole component.